pkg/webview/view.proto

```diff
@@ -119,7 +124,8 @@ message Resource {
   repeated string pending_build_edits = 9;
   google.protobuf.Timestamp pending_build_since = 10;
   bool has_pending_changes = 11;
-  repeated string endpoints = 12;
+  repeated string DEPRECATED_endpoints = 12;
+  repeated Link endpointLinks = 28;
```

Comment:
protobuf fields should be slug cased (endpoint_links)

Reply:
whoop ty